src/fluree/db/peer/http_api.clj

```diff
@@ -833,7 +833,7 @@
                                       (<? (verify-auth db auth authority)))
 
                                     jwt
-                                    (let [jwt-auth (-> (pw-auth/fluree-auth-map (:conn system) jwt) :auth)
+                                    (let [jwt-auth (:auth (pw-auth/fluree-auth-map (:conn system) jwt))
```

Comment:
Totally up to you, but now that I think about it, the other idiomatic way to do this would be:

```clojure
(-> system
    :conn
    (pw-auth/fluree-auth-map jwt)
    :auth)
```